tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/FlatJSDependency.scala

```diff
@@ -7,11 +7,15 @@ import org.scalajs.core.ir.Trees.isValidIdentifier
  */
 final class FlatJSDependency(
     val origin: Origin,
-    val resourceName: String,
+    @deprecatedName('resourceName)
+    val relPath: String,
```

Comment:
We should somehow document that this `relPath` is not the same than the one in `JSDependency`.